### PR TITLE
Field guide section on Delta Scuti pulsators

### PR DESCRIPTION
**Pull request by mcoughlin**

### CODE REVIEW

File: doc/field_guide__delta_scuti.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## Delta Scuti (dscu)
+delta Scuti variables are classic pulsating stars (i.e., they are stars passing through the [instability strip](https://en.wikipedia.org/wiki/Instability_strip)). As these stars expand and contract (i.e. pulsate) their size and temperature changes leading to regular, smooth variations in their brightness. delta Scuti stars are smaller than both Cepheids and RR Lyrae stars, which leads to a short period of oscillations (typically anywhere between 0.03 days and 0.3 days).
```
Comment (janvanroestel) [inline doc/field_guide__delta_scuti.md:2]: delta Scuti variables are classic --> Delta Scuti variables are classic
delta Scuti stars are smaller --> Delta Scuti stars are smaller

File: config.defaults.yaml [hunk 2]
```diff
@@ -51,3 +51,7 @@ docs:
       coordinates: [254.7213, 32.8341]
       period: 0.31087445597147706
```
Comment (dmitryduev) [inline config.defaults.yaml:52]: Do you want to take a stab at the exceedingly-precise periods for other types of objects?

**Review by janvanroestel — APPROVED** [review]
Mayba capitalise Delta Scuti, but look good otherwise

File: config.defaults.yaml [hunk 3]
```diff
@@ -51,3 +51,7 @@ docs:
...
       title: RR Lyrae c
+    delta_scuti:
+      coordinates: [271.6835652500, 30.0281826000]
```
Comment (dmitryduev) [inline config.defaults.yaml:55]: drop trailing zeros?

File: data/golden/delta_scuti.csv [hunk 4]
```diff
@@ -0,0 +1,101 @@
+ra,dec
+99.9554545000,66.9842884000
```
Comment (dmitryduev) [inline data/golden/delta_scuti.csv:2]: drop trailing zeros?

**Review by dmitryduev — APPROVED** [review]
LGTM, just a couple minor qs.

File: data/golden/delta_scuti.csv (hunk 4 above)
Comment (mcoughlin) [reply data/golden/delta_scuti.csv]: done.

File: config.defaults.yaml (hunk 3 above)
Comment (mcoughlin) [reply config.defaults.yaml]: done.

File: config.defaults.yaml (hunk 2 above)
Comment (mcoughlin) [reply config.defaults.yaml:52]: I am less clear what you mean here ;)

Comment (mcoughlin) [reply config.defaults.yaml:52]: I bet the answer is no though!

File: doc/field_guide__delta_scuti.md (hunk 1 above)
Comment (mcoughlin) [reply doc/field_guide__delta_scuti.md]: done

File: config.defaults.yaml (hunk 2 above)
Comment (dmitryduev) [reply config.defaults.yaml:52]: @AshishMahabal brought it up elsewhere. I guess it's fine as is.

File: config.defaults.yaml (hunk 3 above)
Comment (AshishMahabal) [reply config.defaults.yaml]: I would actually round it off to 5 decimal places.
The period too should have a limited number of digits.

**Comment by doccosmos** [comment]
I think we scan periods to 2 min (?) which ~0.001 days and our basic timing is only accurate to 0.1s? So 5 dp should be enough.

— Matthew

**Pull request merged**